public/src/utils.js

```diff
@@ -21,6 +23,7 @@
 			return (diff[0] * 1e3) + (diff[1] / 1e6);
 		};
 	} else {
+		isBrowser = true;
```

Comment:
Pass isBrowser as an argument to the factory function, like XRegExp and console are.